strings/tcg/set_one.json

```
@@ -719,22 +719,22 @@
 			"id": "lawyer",
 			"name": "Lawyer",
 			"desc": "Nanotrasen knows the value of a good lawyer. That's why they're all working hard at our home offices defending us from frivolous labor suits from lazy no-good employees who should be working hard instead of slacking off reading trading cards.",
-			"rules": "When an opponent attacks with a creature with 3 or more power, this card gains {$Taunt}.",
+			"rules": "When an opponent attacks with a creature with 3 or less power, this card gains {$Taunt}.",
 			"icon_state": "lawyer",
 			"power": "0",
 			"resolve": "4",
 			"faction": "Service",
 			"summoncost": "2",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Employee",
+			"cardsubtype": "Service Security",
 			"rarity": "common",
 			"summon_icon_state": "Lawyer"
 		},
 		{
 			"id": "legion",
 			"name": "Legion",
 			"desc": "They are the cursed, damned souls of civilizations born and lost in the flames of Indecipheres, conglomerated into a lump of emaciated bodies, wandering the realms they used to rule... or something along those lines, anyway.",
-			"rules": "When Legion is destroyed, search your deck for a human card, and summon it to the battlefield. Shuffle your deck afterward.",
+			"rules": "When Legion is destroyed, search your deck for a creature card cosing 3 or less and add it to your hand. Shuffle your deck afterward.",
```

```suggestion
			"rules": "When Legion is destroyed, search your deck for a creature card costing 3 or less and add it to your hand. Shuffle your deck afterward.",
```
"**with a cost** of 3 or less" sounds better though typo aside